<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSceneFilterWidget</class>
 <widget class="qSlicerWidget" name="qSlicerSceneFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>25</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Filter scene by node's attribute</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="SceneFilterDomainComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Apply Domain filter</string>
     </property>
     <item>
      <property name="text">
       <string>ANY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TVD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TVDSS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TWT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OWT</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="SceneFilterAttributeLineEdit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Node's attribute to filter</string>
     </property>
     <property name="placeholderText">
      <string>Attribute...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="SceneFilterValueLineEdit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Node's attribute value to filter</string>
     </property>
     <property name="placeholderText">
      <string>Value...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SceneFilterAttributeLineEdit</tabstop>
  <tabstop>SceneFilterValueLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
